tests/diff_context_test.py

```diff
@@ -6,7 +6,10 @@
 
 from mentat.diff_context import DiffContext
 
-rel_path = Path("multifile_calculator/operations.py")
+
+@pytest.fixture
+def abs_path(temp_testbed):
+    return Path(temp_testbed).resolve() / "multifile_calculator/operations.py"
```

Comment:
I feel like this shouldn't be a fixture; how about we just calculate it in each test? Also, temp_testbed is already absolute, so no need to resolve it